.github/workflows/build-and-test-pr.yaml

```
@@ -36,6 +36,14 @@ jobs:
           target-packages: ${{ steps.get-modified-packages.outputs.modified-packages }}
           build-depends-repos: build_depends.repos
 
+      - name: Upload coverage to Codecov
+        if: ${{ steps.get-modified-packages.outputs.modified-packages != '' }}
+        uses: codecov/codecov-action@v2
+        with:
+          files: lcov/total_coverage.info,coveragepy/.coverage
```

[Question] Can we write like this?

```suggestion
          files: |
            lcov/total_coverage.info
            coveragepy/.coverage
```

---

According to the documentation, `files` argument is comma-separated. 
The line break style you suggested is the same as the one separated by a space.
Therefore, I'm not sure if it will work.

> Comma-separated paths to the coverage report(s)
https://github.com/codecov/codecov-action#arguments

---

I see, thank you.
But why is it `Optional`? What if we remove it?

---

Yes, it is a bit strange.
`codecov-action` does not have a default value for the `files` option.
https://github.com/codecov/codecov-action/blob/master/src/buildExec.ts#L96

Also, `codecov-action` is a wrapper of `codecov-uploader`. According to the documentation, the `-f` option is for specifying the file(s) to upload. Although it is not mentioned in the documentation, `codecov-uploader` is supposed to upload all files if we don't set the `-f` option.
https://docs.codecov.com/docs/codecov-uploader#uploader-command-line-arguments

---

Hmm, it's difficult... Thank you for the detailed information!